caddy/caddymain/run.go

```diff
@@ -392,35 +392,53 @@ func initTelemetry() error {
 	}
 
 	// parse and check the list of disabled metrics
-	var disabledMetricsSlice []string
+	var fixedDisabledMetrics []string
```

Comment:
I'm not sure this variable name needs to be changed, the previous name made more sense I think.

Reply:
Yes, I had restore previous name.